.github/buildomat/jobs/build-and-test.sh

```diff
@@ -33,6 +33,12 @@ export RUSTFLAGS="-D warnings"
 export RUSTDOCFLAGS="-D warnings"
 ptime -m cargo +'nightly-2021-11-24' build --locked --all-targets --verbose
 
+#
+# Check that building individual packages as when deploying Omicron succeeds
+#
+banner deploy-check
+ptime -m cargo +'nightly-2021-11-24' run --bin omicron-package -- check
```

Comment:
I know we're doing this above for the build... but is it necessary to specify the toolchain here? Won't it be covered by `rust-toolchain.toml`?

Reply:
I agree it should be. I will remove it here and verify that it uses the expected toolchain from the file.